test/scenarios/fork/ynETHWithdrawals.t.sol

```diff
@@ -0,0 +1,544 @@
+        }
+
+        uint256 _ynETHBalanceBefore = yneth.balanceOf(user);
+        assertGe(_ynETHBalanceBefore, _amount, "testRequestWithdrawal: E1");
+
+        uint256 _expectedAmountOut = yneth.previewRedeem(_expectedTokenAmount);
+        uint256 _queueManagerBalanceBefore = yneth.balanceOf(address(ynETHWithdrawalQueueManager));
+        uint256 _pendingRequestedRedemptionAmountBefore = ynETHWithdrawalQueueManager.pendingRequestedRedemptionAmount();
+
+        vm.startPrank(user);
+        yneth.approve(address(ynETHWithdrawalQueueManager), _expectedTokenAmount);
+        tokenId = ynETHWithdrawalQueueManager.requestWithdrawal(_expectedTokenAmount);
+        vm.stopPrank();
+
+        amountOut = ynETHWithdrawalQueueManager.pendingRequestedRedemptionAmount() - _pendingRequestedRedemptionAmountBefore;
+
+        assertEq(yneth.balanceOf(user), 0, "testRequestWithdrawal: E2");
+        assertGt(ynETHWithdrawalQueueManager.balanceOf(user), 0, "testRequestWithdrawal: E3");
+        assertEq(ynETHWithdrawalQueueManager.ownerOf(tokenId), user, "testRequestWithdrawal: E4");
+        assertEq(yneth.balanceOf(address(ynETHWithdrawalQueueManager)) - _queueManagerBalanceBefore, _expectedTokenAmount, "testRequestWithdrawal: E5");
+        assertApproxEqAbs(amountOut, _expectedAmountOut, 100_000, "testRequestWithdrawal: E6");
+        assertGe(_expectedAmountOut, amountOut, "testRequestWithdrawal: E7");
+
+        {
+            IWithdrawalQueueManager.WithdrawalRequest memory _request = ynETHWithdrawalQueueManager.withdrawalRequest(tokenId);
+            assertEq(_request.amount, _expectedTokenAmount, "testRequestWithdrawal: E8");
+            assertEq(ynETHWithdrawalQueueManager.withdrawalFee(), _request.feeAtRequestTime, "testRequestWithdrawal: E9");
+            assertEq(yneth.previewRedeem(1 ether), _request.redemptionRateAtRequestTime, "testRequestWithdrawal: E10");
+            assertEq(_request.creationTimestamp, block.timestamp, "testRequestWithdrawal: E11");
+            assertEq(_request.creationBlock, block.number, "testRequestWithdrawal: E12");
+            assertTrue(!_request.processed, "testRequestWithdrawal: E13");
+        }
+
+        {
+            uint256 _timestamp = block.timestamp;
+            vm.warp(block.timestamp + ynETHWithdrawalQueueManager.secondsToFinalization());
+            vm.prank(user);
+            // vm.expectRevert(abi.encodeWithSelector(InsufficientBalance.selector, address(ynETHWithdrawalQueueManager).balance, _amount));
+            // fails with above revert reason, but _amount is a little less because of precision loss
+            // ynETHRedemptionAssetsVaultInstance.availableRedemptionAssets(address(0));
+            if (ynETHRedemptionAssetsVaultInstance.availableRedemptionAssets(address(0)) < _amount) {
+                vm.expectRevert();
+                ynETHWithdrawalQueueManager.claimWithdrawal(tokenId, receiver);
```

Comment:
this verification ere related to claimWithdrawal does not belong in this test, and also the revert should be written explicitly 

with selector and params. 

We don't want to use empty vm.expectRevert()

Reply:
fixed in https://github.com/yieldnest/yieldnest-protocol/pull/124/commits/6bf949f723b721a6eac02e32dbbf63fae562bbf5.